### PR TITLE
Added WG Charter

### PR DESCRIPTION
**Pull request by mcollina**
Adds the WG charter.

Fixes https://github.com/nodejs/undici/issues/580

### CODE REVIEW

**Comment by mcollina** [comment]
cc @nodejs/tsc

**Review by ronag — APPROVED** [review]

**Review by dnlup — APPROVED** [review]

**Comment by mcollina** [comment]
Don't land as it needs to wait for the TSC.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+### Undici Working Group
+
+The Node.js Undici is governed by a Working Group
```
Comment (targos) [inline GOVERNANCE.md:3]: Isn't something missing after "The Node.js Undici" ?

Comment (mcollina) [reply GOVERNANCE.md]: Fixed, good spot!

**Review by mhdawson — APPROVED** [review]
LGTM

**Comment by mhdawson** [comment]
@mcollina this has probably had enough time for people to object if they were going to. Do you want to move out of draft ?

**Review by mmarchini — APPROVED** [review]

**Pull request merged**